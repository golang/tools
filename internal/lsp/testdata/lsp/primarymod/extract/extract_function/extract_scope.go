package extract

func _() {
	fn0 := 1
	a := fn0 //@extractfunc("a", "fn0")
}

func fn1() int {
	return 1
}
