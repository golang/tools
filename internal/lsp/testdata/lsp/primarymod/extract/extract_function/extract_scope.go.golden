-- functionextraction_extract_scope_5_2 --
package extract

func _() {
	fn0 := 1
	fn2(fn0) //@extractfunc("a", "fn0")
}

func fn2(fn0 int) {
	a := fn0
}

func fn1() int {
	return 1
}

